scanner/datastore/postgres/distributions.go

```diff
@@ -44,3 +58,15 @@ func (m *matcherStore) Distributions(ctx context.Context) ([]claircore.Distribut
 
 	return dists, nil
 }
+
+func rhelDist(repoName string) (claircore.Distribution, error) {
+	m := rhelCPE.FindStringSubmatch(repoName)
+	if len(m) != 2 {
+		return claircore.Distribution{}, fmt.Errorf("unexpected repo name: %s", repoName)
+	}
+	return claircore.Distribution{
+		DID:       "rhel",
+		VersionID: m[1],
+		Version:   m[1],
+	}, nil
```

Comment:
How will this be different from the `dist_version_id` and `dist_version` fields already in the database currently? Seems like the load of vex data is no longer populating these fields. Should that addressed at load time? this seems like a 'hack'.

```
    dist_id    | dist_version_id |  dist_version  |                            repo_name                             
---------------+-----------------+----------------+------------------------------------------------------------------
 rhel          | 8               | 8              | cpe:/a:redhat:enterprise_linux:8
 rhel          | 9               | 9              | cpe:/a:redhat:enterprise_linux:9::appstream
...

```

Reply:
What I'm doing is essentially a "hack" to account for these now unpopulated columns for Red Hat vulnerabilities. Claircore does not require `dist_version_id` nor `dist_version` when matching vulnerabilities against RHEL images, as it relies on the CPEs, instead. So, they don't need to know the distribution. Plus the data isn't really given to us in a distribution-specific manner, anyway, so this helps us continue to identify RHEL versions for which we have vulnerabilities without having to hardcode anything. Based on my findings, we can find all the current versions of RHEL we care about and more (older ones) with this query.